eventstore/whatsapp_actions.py

```diff
@@ -23,7 +23,8 @@ def handle_operator_message(message):
     return chain(
         get_engage_inbound_and_reply.s(),
         async_create_flow_start.s(
-            flow=settings.RAPIDPRO_OPERATOR_REPLY_FLOW, urns=[f"tel:{msisdn}"]
+            flow=settings.RAPIDPRO_OPERATOR_REPLY_FLOW,
+            urns=[f"whatsapp:{msisdn.lstrip('+')}"],
```

Comment:
Can't we just use the `whatsapp_contact_id` from above?

Reply:
I did a manual request, looks like `message._vnd.v1.chat.owner` is in the `+27820001001` format, so we would still have to strip the `+`. Using the normalised `msisdn` means that if this changes in the future, we'll still be okay

Reply:
Awesome!